inputFiles/hydraulicFracturing/kgdToughness_Storage_Dominated_base.xml

```diff
@@ -0,0 +1,242 @@
+<?xml version="1.0" ?>
+
+<Problem>
+  <Solvers
+    gravityVector="{ 0.0, 0.0, -0.0 }">
+    <Hydrofracture
+      name="hydrofracture"
+      solidSolverName="lagsolve"
+      flowSolverName="SinglePhaseFlow"
+      surfaceGeneratorName="SurfaceGen"
+      logLevel="1"
+      targetRegions="{ Domain, Fracture }"
+      isMatrixPoroelastic="0"
+      useQuasiNewton="1"
+      maxNumResolves="2">
+
+      <NonlinearSolverParameters
+        newtonTol="1.0e-6"
+        newtonMaxIter="20"
+        maxAllowedResidualNorm="1.0e30"
+	lineSearchAction="None"
+      />
+
+      <LinearSolverParameters
+        directParallel="0"/>
+    </Hydrofracture>
+
+    <SolidMechanicsLagrangianFEM
+      name="lagsolve"
+      timeIntegrationOption="QuasiStatic"
+      discretization="FE1"
+      targetRegions="{ Domain, Fracture }"
+      contactRelationName="fractureContact"
+      contactPenaltyStiffness="1.0e0"
+    />
+
+    <SinglePhaseFVM
+      name="SinglePhaseFlow"
+      discretization="singlePhaseTPFA"
+      targetRegions="{ Domain, Fracture }"/>
+
+    <SurfaceGenerator
+      name="SurfaceGen"
+      targetRegions="{ Domain }"
+      nodeBasedSIF="1"
+      initialRockToughness="1e6"
+      isPoroelastic="0"
+      mpiCommOrder="1"/>
+  </Solvers>
+
+  <NumericalMethods>
+    <FiniteElements>
+      <FiniteElementSpace
+        name="FE1"
+        order="1"/>
+    </FiniteElements>
+
+    <FiniteVolume>
+      <TwoPointFluxApproximation
+        name="singlePhaseTPFA"
+	meanPermCoefficient="0.0"
+	/>
+    </FiniteVolume>
+  </NumericalMethods>
+
+  <ElementRegions>
+    <CellElementRegion
+      name="Domain"
+      cellBlocks="{ cb1 }"
+      materialList="{ water, porousRock }"/>
+
+    <SurfaceElementRegion
+      name="Fracture"
+      defaultAperture="0.02e-5"
+      materialList="{ water, rock, fractureFilling, fractureContact, hApertureModel}"/>
+  </ElementRegions>
+
+  <Constitutive>
+    <CompressibleSinglePhaseFluid
+      name="water"
+      defaultDensity="1000"
+      defaultViscosity="1.0e-6"
+      referencePressure="0.0"
+      compressibility="5e-10"
+      referenceViscosity="1.0e-6"
+      viscosibility="0.0"/>
+
+    <PorousElasticIsotropic
+      name="porousRock"
+      solidModelName="rock"
+      porosityModelName="rockPorosity"
+      permeabilityModelName="rockPerm"/>  
+
+    <ElasticIsotropic
+      name="rock"
+      defaultDensity="2700"
+      defaultYoungModulus="30.0e9"
+      defaultPoissonRatio="0.25"/>
+
+    <BiotPorosity
+      name="rockPorosity"
+      defaultGrainBulkModulus="20e9"
+      defaultReferencePorosity="0.3"/>
+
+    <ConstantPermeability
+      name="rockPerm"
+      permeabilityComponents="{ 1.0e-20, 0.0e-18, 0.0e-18 }"/>  
```

Comment:
Just a thought here. Do you think if it will be better and more flexible to define "transversalPermeability = 1e-20" in `ParallelPlatesPermeability`, so it can also work for an inclined kgd fracture (not aligned with the coordinate)? And we probably need to make  `meanPermCoefficient = 1` in `TwoPointFluxApproximation`.